src/signal.rs

```diff
@@ -1654,7 +1788,7 @@ impl<S> Signal for ClipAmp<S>
 impl<S> Bus<S>
     where S: Signal,
 {
-    fn new(signal: S, frames_read: BTreeMap<usize, usize>) -> Self {
+    pub fn new(signal: S, frames_read: BTreeMap<usize, usize>) -> Self {
```

Comment:
Hmmm I left this private intentionally as there is already the `Signall::bus` method for creating a `Bus` around a `Signal`. Is there a reason in particular you'd like this to be public?